### ISSUE TITLE
Allow to move a user to a room with keyboard to help blind people

### ISSUE THREAD

**Issue by alexarnaud** [mumble-voip/mumble#4642]
**Context**
All blind people rely on keyboard shortcuts because they can't use the mouse.

**Describe the feature you have in mind**
To move a user from one channel to another, you'll do :
1. The behaviour I propose here is the same used in the Thunderbird mail client to move message from one folder to another with the keyboard.
2. You have for example the following room in your Mumble server
2.1 Channel 1
2.2 Channel 2, with sub-channel 3 and 4
3. Place the focus on an user with the keyboard or the mouse.
5. Open the contextual menu where you'll find a move item.
6. Select move and press enter to open a sub-menu containing all the top rooms. Here channel 1 and 2.
7. Two possibilities:
7.1 Select channel 1 to move the user to channel 1.
7.2 Select channel 2 to open the channel 2 sub-menu where you'll find this 3 items, channel 2, channel 3 or channel 4.

**Additional context**
Without this, it's impossible for a blind user to move another user to a dedicated room because he can't use drag and drop.

**Comment by Hartmnt** [comment]
As far as I can tell, this is a duplicate of #2322

**Comment by Krzmbrzl** [comment]
Correct - However, as the present issue has more votes for it, I'll close the original issuebin favor of this one.

**Comment by Krzmbrzl** [comment]
Closing this in favor of #4642 (same issue but with more votes)

**Comment by lilmike** [comment]
This feature would be really nice

Michael.

**Comment by Hartmnt** [comment]
The new stable version of Mumble will add a user context menu entry to "Move To Own Channel" which should be a nice workaround until we implement a proper dialog for this. I will be therefore changing the title of this issue to reflect that.
